.github/workflows/build-and-test.yml

```diff
@@ -3,21 +3,21 @@ name: Build and Test
 on:
   workflow_dispatch:
   push:
-    branches: [ 'main', 'dev', 'feature/*' ]
+    branches: ["main", "dev", "feature/*"]
   pull_request:
-    branches: [ 'main', 'dev' ]
+    branches: ["main", "dev"]
 
 jobs:
   build-and-test:
-    runs-on: windows-latest # Temporary switch to windows till https://github.com/microsoft/vstest/issues/4549 is released in dotnet version after 7.0.400
+    runs-on: windows-latest
```

Comment:
With the release we can probably switch back to linux runners for faster build time now.